src/lib/link.ts

```diff
@@ -520,6 +521,39 @@ export class Link {
     return height;
   }
 
+  public async timeoutPackets(
+    source: Side,
+    packets: readonly PacketWithMetadata[]
+  ): Promise<readonly MsgResult[]> {
```

Comment:
I wouldn't really return all results. It fails if the tx fails.
Maybe return the block height where they were submitted? Or nothing until you find a need

Reply:
Great, I originally returned nothing here, and then switched to this but it's premature optimisation.